fanuc_lrmate200ic5h_moveit_config/launch/move_group.launch

```diff
@@ -5,15 +5,16 @@
   <!-- GDB Debug Option -->
   <arg name="debug" default="false" />
   <arg unless="$(arg debug)" name="launch_prefix" value="" />
-  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
+  <arg     if="$(arg debug)" name="launch_prefix"
+	   value="gdb -x $(find fanuc_lrmate200ic5h_moveit_config)/launch/gdb_settings.gdb --ex run --args" />
```

Comment:
Or at least: I've not had issues starting these launch files without that file.

But I don't know whether I've actually ran the MoveIt pkgs in debug mode though.